lib/flutter_webview_plugin.dart

```diff
@@ -90,6 +142,10 @@ class FlutterWebviewPlugin {
     await _channel.invokeMethod('launch', args);
   }
 
+  Future<String> evalJavascript(String code) {
```

Comment:
IOS implemented
Android not implemented

Reply:
OK